kpi-tracker/src/main.rs

```diff
@@ -209,48 +249,85 @@ fn get_account_transaction_details(
 fn to_block_events(block_hash: BlockHash, block_item: BlockItemSummary) -> Vec<BlockEvent> {
     let mut events: Vec<BlockEvent> = Vec::new();
 
-    match block_item.details {
+    match &block_item.details {
         AccountTransaction(atd) => {
-            let details = get_account_transaction_details(&atd, block_hash);
-            println!(
-                "TRANSACTION:\nhash: {}\ndetails: {:?}",
-                block_item.hash, &details
-            ); // Logger debug
-            let event = BlockEvent::AccountTransaction(block_item.hash, details);
+            let details = get_account_transaction_details(atd, block_hash);
+            let affected_accounts: Vec<TransactionAccountRelation> = block_item
+                .affected_addresses()
+                .into_iter()
+                .map(|address| {
+                    TransactionAccountRelation(
+                        CanonicalAccountAddress::from(address),
+                        block_item.hash,
+                    )
+                })
+                .collect();
+
+            let affected_contracts: Vec<TransactionContractRelation> = block_item
+                .affected_contracts()
+                .into_iter()
+                .map(|address| TransactionContractRelation(address, block_item.hash))
+                .collect();
+
+            log::debug!(
+                "Account transaction found: {}\n{:?}\n{}\n{}",
+                block_item.hash,
+                details,
+                &affected_accounts
+                    .iter()
+                    .map(|account_relation| format!("{:?}", account_relation))
+                    .collect::<Vec<_>>()
+                    .join("\n"),
+                &affected_contracts
+                    .iter()
+                    .map(|contract_relation| format!("{:?}", contract_relation))
+                    .collect::<Vec<_>>()
+                    .join("\n")
```

Comment:
Also this is way too much logging to survive for the final version. I assume it will be removed?

Reply:
What logging level would the production service be configured with, do you reckon? Could move all the details of the entities to `trace` along with the affected accounts/contracts?

Reply:
Probably at info we should just log normal progress, i.e., "processed block" and such.

I don't think these details you are logging here should be logged at all. We will see this in the database.

Reply:
I don't think these details would lhelp debug an issue for instance.

Reply:
I'll remove them!